### PR TITLE
Node version

### PR DESCRIPTION
**Pull request by carvilsi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Created by .ignore support plugin (hsz.mobi)
 .DS_Store
-.idea/
-target/
-*.iml
\ No newline at end of file
+files/
+node_modules
+lib/
```
Comment (gvdent) [inline .gitignore:4]: lib/ is in the .gitignore but also still in the repo?

File: config/custom-environment-variables.coffee [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  services:
+    projectController:
+      endpoint:  "SERVICES_PROJECT_ENDPOINT"
+    projectDatabase:
+      endpoint:  "SERVICES_DATABASE_ENDPOINT"
+    systemDatabase:
+      endpoint:  "SERVICES_SYSTEM_DATABASE_ENDPOINT"
+    flock:
+      endpoint:  "SERVICES_FLOCK_ENDPOINT"
```
Comment (gvdent) [inline config/custom-environment-variables.coffee]: By the looks of it these are based on the weaver-server custom env variables. Please delete everything besides the server.port and logging.* items.

File: config/local.coffee [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# This file resides in the git repository but do not track it in git by using the following command:
+# git update-index --assume-unchanged config/local.coffee
+#
+# Use it for custom local configuration here to override default configurations
+module.exports =
+  application:
+    sounds:
+      muteAll: false
+  logging:
+    console: 'silly'
```
Comment (gvdent) [inline config/local.coffee:10]: That is silly indeed ;)

File: doc/api/index.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Weaver-server-virtuoso</title>
```
Comment (gvdent) [inline doc/api/index.html]: Weaver-server-virtuoso?

File: doc/api/swagger.yml [hunk 5]
```diff
@@ -0,0 +1,141 @@
+# this is an example of the Uber API
```
Comment (gvdent) [inline doc/api/swagger.yml]: This looks great! 👍

**Review by gvdent — CHANGES_REQUESTED** [review]
Minor comments

File: doc/api/index.html (hunk 4 above)
Comment (carvilsi) [reply doc/api/index.html]: oooops

File: config/custom-environment-variables.coffee (hunk 2 above)
Comment (carvilsi) [reply config/custom-environment-variables.coffee]: 👍

File: config/local.coffee (hunk 3 above)
Comment (carvilsi) [reply config/local.coffee:10]: -.-

**Review by gvdent — APPROVED** [review]

**Pull request merged**